lib/experiments-page.php

```diff
@@ -246,6 +246,19 @@ function gutenberg_initialize_experiments_settings() {
 		)
 	);
 
+	// create a new experiment for hiding blocks based on screen size
+	add_settings_field(
+		'gutenberg-hide-blocks-based-on-screen-size',
+		__( 'Hide blocks based on screen size', 'gutenberg' ),
+		'gutenberg_display_experiment_field',
+		'gutenberg-experiments',
+		'gutenberg_experiments_section',
+		array(
+			'label' => __( 'Extends block visiblity block supports to hide blocks based on screen size.', 'gutenberg' ),
```

Comment:
I think the title/description could be improved. It makes it sound like blocks are automatically hidden based on screen size without the user doing anything, but I think what it really will do is provide responsive design controls for hiding blocks.

Reply:
True indeed! I'll update